gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/adaptive/clickhouse/ClickHouseAdaptiveQueryExecSuite.scala

```diff
@@ -42,7 +43,8 @@ class ClickHouseAdaptiveQueryExecSuite extends AdaptiveQueryExecSuite with Glute
 
   override def sparkConf: SparkConf = {
     super.sparkConf
-      .set("spark.gluten.sql.columnar.forceShuffledHashJoin", "false")
+      .set(GlutenConfig.COLUMNAR_FORCE_SHUFFLED_HASH_JOIN_ENABLED.key, "false")
+      .set(GlutenConfig.NOOP_WRITER_ENABLED.key, "false")
```

Comment:
The following test will report an error, as `GlutenNoopWriterRule` will add a `FakeRowAdaptor` node, which will cause the test check to fail, thus we default false here.
```
SPARK-30953: InsertAdaptiveSparkPlan should apply AQE on child plan of v2 write commands
```